docs/user/tutorials/01-10-simple-usage.md

```diff
@@ -0,0 +1,81 @@
+# Use Docker Registry
+
+This tutorial shows how you can push an image to the Docker Registry and use it.
+
+## Prerequsities
+
+* [Kyma CLI v3](https://github.com/kyma-project/cli)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/)
+* [Docker](https://www.docker.com/)
+
+## Steps
+
+1. Create a `simple.py` file, and paste the following content:
+
+   ```python
+   import time
+   import datetime
+    
+   while True:
+     print("Current time:", datetime.datetime.now(), flush=True)
+     time.sleep(10)
+   ```
+
+2. Create `Dockerfile` with the following content:
+
+   ```dockerfile
+   FROM python:3-alpine
+
+   WORKDIR /usr/src/app
+
+   COPY simple.py .
+   
+   CMD [ "python", "./simple.py" ]
+   ```
+
+3. Build and push the image to Docker Registry:
+
+   ```bash
+   docker build -t simple-image .
+   ```
+
+4. Import the image to Docker Registry:
+
+    <!-- TODO: presiquites: kyma cli (new version) -->
```

Comment:
I think this comment is no longer needed

Reply:
removed